extensions/cli/debug/src/main/java/mil/nga/giat/geowave/cli/debug/FullTableScan.java

```diff
@@ -63,4 +64,11 @@ protected long runQuery(
 		return count;
 	}
 
+	@Override
+	public Object computeResults(
+			OperationParams params ) {
+		// TODO Auto-generated method stub
+		return null;
```

Comment:
problem (albeit merely debug commands, but still seems relatively straightforward to fix)